pracownia_projektowa/prezentacja_prototypu/prezentacja_prototypu.tex

```diff
@@ -0,0 +1,245 @@
+
+Wydział Informatyki, Elektroniki i Telekomunikacji
+
+Katedra Informatyki 
+\newline \newline
+Projekt realizowany pod opieką \\dr. hab. inż. Piotra Faliszewskiego
+
+}
+\date{6 czerwca 2016}
+
+%----------------------------------------------------------------------------------------
+
+\begin{document}
+
+\frame{\titlepage}
+
+\begin{frame}
+\frametitle{Plan}
+\tableofcontents
+\end{frame}
+
+%----------------------------------------------------------------------------------------
+%	PRESENTATION SLIDES
+%----------------------------------------------------------------------------------------
+
+%------------------------------------------------
+\section{Rejestracja}
+%------------------------------------------------
+\subsection{Rejestracja}
+%------------------------------------------------
+\begin{frame}
+\frametitle{Dane użytkownika}
+\includegraphics[height=0.63\paperheight]{screenshots/01_logowanie.png}
+\end{frame}
+%------------------------------------------------
+\begin{frame}
+\frametitle{Dane użytkownika}
+Są zabezpieczone dzięki dostarczonym przez django mechanizmom zarządzania kontem i poufnymi danymi użytkownika.
+\end{frame}
+
+%------------------------------------------------
+\section{Tworzenie wyborów}
+\begin{frame}
+\frametitle{Panel dodawania wyborów}
+\begin{center}
+\includegraphics[width=0.8\paperwidth]{screenshots/04_new_elections.png}
+\end{center}
+\end{frame}
+%------------------------------------------------
+\begin{frame}
+\frametitle{Dodawanie wyborów}
+\begin{itemize}
+	\item Z pliku
+	\item Z rozkładu normalnego
+	\item ?
+\end{itemize}
+\begin{center}
+	\includegraphics[width=0.8\paperwidth]{screenshots/06_load_soc.png}
+\end{center}
+\end{frame}
+%------------------------------------------------
+
+\subsection{Pobieranie danych z pliku}
+\begin{frame}
+\frametitle{Pobieranie danych z pliku}
+\includegraphics[width=0.8\paperwidth]{screenshots/05_file_elections_01.png}
+\end{frame}
+
+
+%------------------------------------------------
+\begin{frame}
+\frametitle{Dane gotowe do obliczania wyborów}
+\includegraphics[width=0.8\paperwidth]{screenshots/10_soc_file_loaded_other.png}
+\end{frame}
+
+%------------------------------------------------
+\subsection{Generowanie z rokładu normalnego}
+\begin{frame}
+\frametitle{Ustalanie parametrów populacji}
+\includegraphics[width=0.8\paperwidth]{screenshots/16_normalny.png}
+\end{frame}
+%------------------------------------------------
+\begin{frame}
+\frametitle{Ustalanie parametrów populacji...}
+\includegraphics[height=0.5\paperheight]{screenshots/17_normalny_limit.png}
+\begin{itemize}
+\item ograniczenia na ilość wierszy w bazie danych
+\item brak możliwości określenia timeoutu...
+\end{itemize}
+\end{frame}
+%------------------------------------------------
+\begin{frame}
+\frametitle{Wykres}
+\includegraphics[height=0.5\paperheight]{screenshots/18_wykres.png}
+\begin{itemize}
+\item dodatkowe informacje o punkcie
+\item inne sposoby generowania wykresów...
```

Comment:
to jest takie TODO? trochę nie wiadomo o co chodzi (w sumie ja nadal nie wiem xd)